content/id/docs/tasks/configure-pod-container/share-process-namespace.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Pembagian Namespace Process antara Kontainer-Kontainer pada sebuah Pod
+min-kubernetes-server-version: v1.10
+content_type: task
+weight: 160
+---
+
+<!-- overview -->
+
+{{< feature-state state="stable" for_k8s_version="v1.17" >}}
+
+Dokumen ini akan menjelaskan menkanisme konfigurasi pembagian namespace
+process dalam sebuah Pod. Ketika pembagian namespace process diaktifkan untuk sebuah Pod,
+proses yang ada di dalam kontainer akan bersifat transparan pada semua kontainer
```

Comment:
nit: we now use [Container](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#container-v1-core) instead of kontainer, same comment applies for all occurence in this page
```suggestion
proses yang ada di dalam Container akan bersifat transparan pada semua Container
```